pybossa/api/task_run.py

```diff
@@ -84,7 +85,7 @@ def _preprocess_post_data(self, data):
             data['info'] = {}
             pyb_answer_url = upload_json_data(json_data=info, upload_path=path,
                 file_name='pyb_answer.json', encryption=with_encryption,
-                conn_name='S3_TASKRUN', upload_root_dir=upload_root_dir
+                conn_name=conn_name, upload_root_dir=upload_root_dir
```

Comment:
Should a copy be made to both buckets for a period of time (6 months or a year) until final migration to v2?

Reply:
As we're upgrading staging buckets to v2 and taskrun payloads contains url path referring to the staging bucket path, its not required to make a copy to v1 bucket.